WorkflowTesting/Sources/Internal/RenderTester+TestContext.swift

```diff
@@ -117,6 +123,11 @@
                     producedOutput = output
                 }
             }
+
+            private struct WorkflowKey: Hashable {
+                let type: ObjectIdentifier
```

Comment:
Why `ObjectIdentifier` instead of `Any.Type`?

Reply:
`ObjectIdentifier` gets us synthesized `Equatable` and `Hashable` conformance. Not sure if there's a benefit to using `Any.Type`.